ArcadiaParties.Data.Abstractions/DTOs/UserDTO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ArcadiaParties.Data.Abstractions.DTOs
+{
+    public class UserDTO
+    {
+        public string Identity { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public DateTime BirthDate { get; set; }
+
+        public DateTime HireDate { get; set; }
+
+        public ICollection<UserRoleDTO> UserRoles { get; set; }
```

Comment:
Do you really need the whole role object? I suppose at the moment that it is enough to have just collection of strings to define user roles.

Reply:
DTO models changed